Obsolete Rfc2898DeriveBytes.CryptDeriveKey (#57002)
dotnet/runtime: docs/project/list-of-diagnostics.md

|  __`SYSLIB0030`__ | HMACSHA1 always uses the algorithm implementation provided by the platform. Use a constructor without the useManagedSha1 parameter. |
|  __`SYSLIB0031`__ | EncodeOID is obsolete. Use the ASN.1 functionality provided in System.Formats.Asn1. |
|  __`SYSLIB0032`__ | Recovery from corrupted process state exceptions is not supported; HandleProcessCorruptedStateExceptionsAttribute is ignored. |

> |  __`SYSLIB0033`__ | Rfc2898DeriveBytes.CryptDeriveKey is obsolete and no longer supported. Use PasswordDeriveBytes.CryptDeriveKey instead. |

Reviewer: Nit on the message (I just replaced "no longer" with "is not" in all of the other messages that used that phrase).

```suggestion
|  __`SYSLIB0033`__ | Rfc2898DeriveBytes.CryptDeriveKey is obsolete and is not supported. Use PasswordDeriveBytes.CryptDeriveKey instead. |
```
